### ISSUE TITLE
Training epoch problem.

### ISSUE THREAD

**Issue by Rui-li023** [ymxlzgy/echoscene#11]
Hi, I found that the E.2 Implementation Details section of the echoscene paper on arxiv mentions that it requires training for 2000 epochs. However, I just discovered in the issues section that you mentioned it was only trained for 200 epochs. I would like to know which is the correct value.

**Comment by ymxlzgy** [comment]
Sorry, it is been a long period, and I have corrected the answer in the last issue. I think 2000 is the correct number. The training set contains 5,037 rooms, and the bs was 64 when I trained. The learning rate iterates from 1e-4 at step 0 to 5e-6 at 140,000. So the epochs would be near 140,000 / (5,037 / 64), which is approximately 2,000.
